tests/cookbooks/harness.py

```
@@ -0,0 +1,101 @@
+    if os.path.isfile(_LOCAL_KEYFILE):
+        v = Path(_LOCAL_KEYFILE).read_text().strip()
+        if v:
+            return v
+    try:
+        from dotenv import dotenv_values
+        for p in (_WORKSPACE_ENV, str(Path.cwd() / ".env")):
+            v = dotenv_values(p).get("FISH_API_KEY")
+            if v:
+                return v.strip()
+    except Exception:
+        pass
```

Ruff (0.15.15)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)